content/docs/reference/design-docs/conflict-detection/index.en.md

```diff
@@ -9,131 +9,210 @@ weight: 45
 This document is a work in progress
 {{% /pageinfo %}}
 
-Conflict detection is a process which enables figuring out if a timetable,
-comprised of many train paths, can be performed as planned.
+Conflict detection is the process of looking for timetable conflicts.
+A timetable conflict is any predictable condition which disrupts planned operations.
+Planned operations can be disrupted if a train slows down, is prevented from proceeding, or delayed.
```

Comment:
```suggestion
Planned operations can be disrupted if a train slows down, is prevented from proceeding, or is delayed.
```

Reply:
no need to repeat `is`. it parses as follows: `is (prevented from proceeding || delayed)`

Reply:
The sentence structure and comma placement feel wrong in that case, I believe `X if A, B, or C` is almost always interpreted as `(A | B | C) => X`

What about "Planned operations can be disrupted if a train slows down, i.e. when it is delayed or prevented from proceeding"?

Reply:
"Planned operations can be disrupted if a train is slowed down, prevented from proceeding, or delayed."

If you don't want to repeat the is.

Reply:
> I believe X if A, B, or C is almost always interpreted as (A | B | C) => X

That's the intent

> What about "Planned operations can be disrupted if a train slows down, i.e. when it is delayed or prevented from proceeding"?

It does not have the same meaning:
 - slowing down is for when the train is moving, slows down, but does not stop
 - delayed for when a departure cannot occur at the planned time
 - prevented from proceeding line about when a route is not set in time, and the train has to stop

Reply:
What about this version?

> Planned operations can be disrupted if a train **either** slows down, is prevented from proceeding, or delayed.

I asked for Garance's input, apparently the `is` isn't necessary like I though it was, but it still a bit hard to parse the initial sentence right (which could cause misunderstandings). Adding an `either` makes it clearer.

Reply:
either does not quite work, as it's usually used to express an alternative between *two* options. How do you feel about @Khoyo's version?

Reply:
I do like his version

Reply:
Alright, let's roll with it then